### DISCUSSION TITLE
关于训练超市小票识别模型的问题

### DISCUSSION THREAD

**Discussion by huilunc** [PaddlePaddle/PaddleOCR#12581] [Q&A]
如题，新手需要训练一个检测并识别超市小票的模型，需要识别的只有小票上的总金额和日期。准备的图片真实数据600多张，合成数据1000张。标记用ppocrlabel只标记了小票上的总金额和日期，这样训练检测模型可行吗？还是必须全部文字都做标记？

真实图片都是一张小票放在桌上拍的或者手拿着拍的，照片尺寸很大，基本都是(2000+ x 2000+)，需要resize吗？因为在config里看到训练过程中会自动resize所以不清楚需不需要额外做这个。 以及预处理（灰度，二值化这些）？在此之前我已经训练了目标检测模型检测照片中小票的位置然后剪裁到基本只剩下小票本票作为OCR的训练数据，之所以这样做是因为尝试过单纯用opencv里的方法预处理发现效果不好。

因为识别模型我觉得效果已经很好了所以初步想法是训练检测模型然后直接用现成的识别模型。

新手很多地方不太清楚，感谢帮助！

**Comment by huilunc** [comment]
![IMG_0351](https://github.com/PaddlePaddle/PaddleOCR/assets/116563407/f614e8db-fc5f-4136-a567-27abcae79eef)

![IMG_0356](https://github.com/PaddlePaddle/PaddleOCR/assets/116563407/ffa6898b-aab9-479c-8c5e-997f9a40cb7d)
小票照片如图

**Comment by GreatV** [comment]
可以

**Reply by huilunc** [reply]
感谢！
